samples/bridge_sample/js/bridge_sample.js

```diff
@@ -0,0 +1,32 @@
+/* Copyright 2018-present Samsung Electronics Co., Ltd. and other contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+var bridge = require('bridge');
```

Comment:
var bridge -> var Bridge  
```
var Bridge  = require('bridge');
this.bridge = new Bridge(native.MODULE_NAME);
```
: A constructor function name needs to start with a capital letter.